imdb/movie_rater.py

```diff
@@ -0,0 +1,75 @@
+from BeautifulSoup import BeautifulSoup
+from mechanize import Browser
+import re
+import json
+
+
+def display_all_names(soup, item):
+    all_names = []
+    all_names = soup.findAll('span', itemprop=item)
+    all_names = str(all_names)
+    all_names = BeautifulSoup(all_names)
+    all_names1 = all_names.findAll('span', itemprop="name")
+    str1 = ""
+    for values in all_names1:
+        str1 = str1 + BeautifulSoup(str(values)).find('span').contents[0]+", "
+    return str1
+
+
+def get_soup(movie):
+    movie_search = '+'.join(movie.split())
+    url = "http://www.imdb.com/find?q=" + movie_search + "&s=all"
+    br = Browser()
+    try:
+        br.open(url)
+        link = br.find_link(url_regex=re.compile(r'/title/tt.*'))
+        res = br.follow_link(link)
+    except:
```

Comment:
always catch specific error types explicitly like `catch ValueError` so that unexpected errors won't go unnoticed